utils.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright © 2022 Lisk Foundation
```

Comment:
If this is a new file, probably the date should be set to 2023?

Reply:
Done.